Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/Meter.tsx

```diff
@@ -175,48 +175,21 @@ function Meter(props: IProps) {
-                {tab === 'notes'? <div className={"tab-pane active"} style={{ maxHeight: window.innerHeight - 215 }}>
-                    <NoteWindow ID={props.MeterID} Type='Meter' />
-                </div> : null}
-                {tab === 'meterInfo' ? <div className={"tab-pane active"} id="meterInfo" style={{ maxHeight: window.innerHeight - 215 }}>
-                    <MeterInfoWindow Meter={meter} StateSetter={(meter: OpenXDA.Types.Meter) => setMeter(meter)} />
-                </div> : null}
-                {tab === 'additionalFields' ?<div className={"tab-pane active" } id="additionalFields" style={{ maxHeight: window.innerHeight - 215 }}>
-                    <AdditionalFieldsWindow ID={props.MeterID} Type='Meter' Tab={tab} />
-                </div> : null}
-                {tab === 'substation' ?<div className={"tab-pane active"} id="substation" style={{ maxHeight: window.innerHeight - 215 }}>
-                    <MeterLocationWindow Meter={meter} StateSetter={(meter: OpenXDA.Types.Meter) => setMeter(meter)} />
-                </div> : null}
-                {tab === 'eventChannels' ?<div className={"tab-pane active"} id="eventChannels">
-                    <MeterEventChannelWindow Meter={meter} IsVisible={tab === "eventChannels"} />
-                </div> : null}
-                {tab === 'trendChannels' ?<div className={"tab-pane active"} id="trendChannels">
-                    <MeterTrendChannelWindow Meter={meter} IsVisible={tab === "trendChannels"} />
-                </div> : null}
-                {tab === 'channelScaling' ?<div className={"tab-pane active"} id="channelScaling">
-                    <ChannelScalingWindow Meter={meter} IsVisible={tab === "channelScaling"} />
-                </div> : null}
-                {tab === 'assets'? <div className={"tab-pane active"} id="assets">
-                    <MeterAssetWindow Meter={meter} /> 
-                </div>: null}
-                {tab == "configurationHistory" ?<div className={"tab-pane active" }>
-                    <MeterConfigurationHistoryWindow Meter={meter} />
-                </div> : null}
-                {tab == "dataRescue" ? <div className={"tab-pane active"}>
-                    {dataRescueWindow}
-                </div> : null }
-                {tab == "dataMerge" ?<div className={"tab-pane active"} id="dataMerge">
-                    {dataMergeWindow}
-                </div> : null}
-                {tab == "dataDelete" ?<div className={"tab-pane active"} id="dataDelete">
-                    {dataDeleteWindow}
-                </div> : null}
-                {tab == "maintenance" ?<div className={"tab-pane active"} id="maintenance">
-                    <MeterMaintenanceWindow Meter={meter} />
-                </div> : null}
+            <div className="tab-content" style={{ flex: 1, display: 'flex', flexDirection: 'column', overflow: 'hidden' }}>
+                {tab === 'notes' && <NoteWindow ID={props.MeterID} Type='Meter' />}
```

Comment:
```suggestion
                {tab === 'notes' ? <NoteWindow ID={props.MeterID} Type='Meter' /> : null}
```
we try to not use the && condition here because `<ntoeWindow....` is not a bool it will fail lint.

Reply:
Got it